tests/Sniffs/Exceptions/ExceptionNameEndsWithExceptionSniffTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Consistence\Sniffs\Exceptions;
+
+class ExceptionNameEndsWithExceptionSniffTest extends \Consistence\Sniffs\TestCase
+{
+
+	public function testInvalidClassName()
+	{
+		$resultFile = $this->checkFile(__DIR__ . '/data/InvalidClassName.php');
+
+		$this->assertSniffError(
+			$resultFile,
+			7,
+			ExceptionNameEndsWithExceptionSniff::CODE_NOT_ENDING_WITH_EXCEPTION,
+			'Exception class name "InvalidClassName" must end with "Exception".'
+		);
+	}
+
+	public function testValidClassName()
+	{
+		$resultFile = $this->checkFile(__DIR__ . '/data/ValidNameException.php');
+
+		$this->assertNoSniffError($resultFile, 7);
+	}
+
+	public function testValidClassNameThatExtendsCustomException()
+	{
+		$resultFile = $this->checkFile(__DIR__ . '/data/FooException.php');
```

Comment:
Please rename `FooException` to `ValidClassNameThatExtendsCustomException` to be more descriptive.